src/cache/DefaultCache.php

```diff
@@ -55,7 +61,11 @@ public function set(string $name, $value)
    */
     public function get(string $name)
     {
-        return $this->cache->get($name);
+        if ($this->cache->hasItem($name)) {
```

Comment:
This `hasItem` is likely already doing an internal `get`, so why are we getting it again below? Just call `get`?

Reply:
it doesn't just return string value, it returns object.

Reply:
And?

Reply:
Updated.